test/routes/index-test.js

```diff
@@ -1,19 +1,20 @@
 const { expect } = require('chai');
 const got = require('got');
-
-const { server } = require('../../src/index');
+const App = require('../../src/app');
```

Comment:
IMHO for API and E2E tests, it's better to load your app via the real entry point defined in `package.json` since real users will have to load your app in this way. So let's just `require('../../')` here.